plugins/kubernetes/app/assets/javascripts/kubernetes/controllers/kubernetes_release_wizard_ctrl.js

```diff
@@ -0,0 +1,31 @@
+samson.controller('KubernetesReleaseWizardCtrl', function($scope) {
+  $scope.steps = [
+    {
+      step: 1,
+      current: false
+    },
+    {
+      step: 2,
+      current: false
+    }
+  ];
+
+  $scope.next = function() {
+    showStep($scope.currentStep + 1);
+  };
+
+  $scope.previous = function() {
+    showStep($scope.currentStep - 1);
+  };
+
+  function showStep(step) {
+    _.findWhere($scope.steps, {step: step}).current = true;
```

Comment:
Using a find is probably overkill here since it's just an ordered array of steps.